package.json

```
@@ -23,6 +23,7 @@
     "seed": "yarn rw exec seed"
   },
   "scripts": {
-    "rw-test-e2e": "npx playwright test e2e"
+    "rw-test-e2e": "npx playwright test e2e",
+    "rw-seed": "yarn rw exec seed_products"
```

By default, `yarn rw exec` generates `db` before running the script:
```
dom@evaM1 ~/p/r/e/example-store (feat/product-seeding-script-#15)> yarn rw exec seed_products
yarn run v1.22.17
$ /Users/dom/prjcts/rw/example-store/example-store/node_modules/.bin/rw exec seed_products
[22:57:57] Generating Prisma client [started]
[22:57:57] Generating Prisma client [completed]
[22:57:57] ...
``` 

But looks like you're not using `db` at all. So you can tell it not to generate it with the `--no-prisma` flag:

```suggestion
    "rw-seed": "yarn rw exec seed_products --no-prisma"
```

I also suggested renaming `rw-seed` to `rw-seed-products` just to distinguish it from prisma; let me know if you think that's too long or anything though!